templates/subject.html

```diff
@@ -30,13 +39,17 @@
 {% endblock %}
 
 {% block content %}
-    <table class="table table-hover table-striped">
+
+    <table class="table table-hover table-striped" id="myTable">
     <thead>
       <tr>
-        <th>title</th>
+          <!-- Click "Title" to sort by names, and "population" to sort by country.
```

Comment:
What do you mean by:
> `and "population" to sort by country.`

Reply:
it's a spelling mistake i meant by largest or lowest number of population.